### DISCUSSION TITLE
2.3.90 upgraded to 2.3.100 but no Cases menu.

### DISCUSSION THREAD

**Discussion by UMWP** [Security-Onion-Solutions/securityonion#7145] [General] [answered]
Hello.

SO standalone 2.3.90 upgraded to 2.3.100 - all SO hotfixes, upgrades and OS upgrades done via sudo soup without errors.
System rebooted, all services up and running. However TheHive is still there and no Cases menu. After escalating an alert got: "Request failed with status code 404" on top of the page. Escalated alert doesn't show in TheHive or nowhere else.

Besides there are no predefined dashboards in Grafana (for example dashboard for standalone) which were in version 2.3.90.

Regards
UMWP

**Comment by dougburks** [comment, marked as answer]
> However TheHive is still there and no Cases menu.

It is normal for TheHive to still be there. If you're not seeing the Cases option on the left side of SOC, first try shift-reload in your browser to make sure there are no caching issues. If that doesn't help, then what is the output of the following?
```
sudo docker images
```

**Comment by UMWP** [comment]
You are absolutely right, Sir. It must have been caching issue. Now it's okay. Also predefined dashboards in Grafana are the same as they used to be. Thank You so much for your lightning fast and correct answer. I'm very sorry for bothering You and taking your precious time.

I would also like to especially thank You Sir (and all of your team members) very, very much for such a great project, which SO definitely is.

Best regards
Mirek
